shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -111,8 +111,11 @@
 // |shell::PlatformView|
 void PlatformViewIOS::UpdateSemantics(blink::SemanticsNodeUpdates update,
                                       blink::CustomAccessibilityActionUpdates actions) {
+  FML_DCHECK(owner_controller_);
   if (accessibility_bridge_) {
     accessibility_bridge_->UpdateSemantics(std::move(update), std::move(actions));
+    [[NSNotificationCenter defaultCenter] postNotificationName:@"FlutterSemanticsUpdate"
```

Comment:
Things interested in this notification will need to know the notification name, right?  The name probably should be exposed in a public header.

Reply:
Good point.  I'm not quite sure where to expose that though.  Perhaps on `ensureSemanticsEnabled` in `FlutterViewController`?

Reply:
er - FlutterEngine.h

Reply:
Documenting it is good, but I think adding something like `const NSNotificationName FlutterSemanticsUpdateNotification` would be better.